module/Activity/view/email/options-overdue.phtml

```diff
@@ -14,7 +14,16 @@ Er zijn een aantal opties ouder dan drie weken,<br />
     </tr>
     <?php foreach($options as $option): ?>
         <tr>
-            <td><?= $option->getProposal()->getOrgan() === null ? $option->getProposal()->getCreator()->getMember()->getFullName() : $option->getProposal()->getOrgan()->getAbbr() ?></td>
+            <?php
+            if ($option->getProposal()->getOrgan() !== null) {
+                $creator = $option->getProposal()->getOrgan()->getAbbr();
+            } elseif ($option->getProposal()->getOrganAlt() !== null) {
+                $creator = $option->getProposal()->getOrganAlt();
+            } else {
+                $creator = $option->getProposal()->getCreator()->getMember()->getFullName();
+            }
+            ?>
```

Comment:
Consider refactoring this block of conditions. This block is also used a few lines down on line 52-59. A possible solution is to define a new method in the `ActivityOptionProposal` model which returns the creator.

```php
public function getCreator(): string {
  // Logic to determine the proposal was created by an organ, alt, or a member
  // Whom ever the creator is, always return a string (predictable)
}
```

Comment also applicable to the file `module/Activity/view/activity/activity-calendar/index.phtml`

Reply:
Alternatively, you can put the refactored code in a service. A possible candidate is the ` ActivityCalendar`. Putting the code in a service instead of the model might be better practice in order to keep your models slim (no business logic). Not entirely sure which is best because I'm not very familiar with Zend framework's architecture.

Reply:
As the functionality really only relates to the model, I added a function to the model.